frontend/model/notifications/templates.js

```diff
@@ -233,16 +250,21 @@ export default ({
     }
 
     const bodyTemplateMap = {
-      [PROPOSAL_INVITE_MEMBER]:
-        L("{strong_}{name}'s{_strong} proposal to add {member} to the group was {strong_}{closedWith}{_strong}.", args),
-      [PROPOSAL_REMOVE_MEMBER]:
-        L("{strong_}{name}'s{_strong} proposal to remove {member} from the group was {strong_}{closedWith}{_strong}.", args),
-      [PROPOSAL_GROUP_SETTING_CHANGE]:
-        L("{strong_}{name}'s{_strong} proposal to change group's {setting} to {value} was {strong_}{closedWith}{_strong}.", args),
-      [PROPOSAL_PROPOSAL_SETTING_CHANGE]:
-        L("{strong_}{name}'s{_strong} proposal to change group's {setting} was {strong_}{closedWith}{_strong}.", args),
-      [PROPOSAL_GENERIC]:
-        L('{strong_}{name}\'s{_strong} proposal "{title}" was {strong_}{closedWith}{_strong}.', args)
+      [PROPOSAL_INVITE_MEMBER]: isCreator
+        ? L('your proposal to add {member} to the group was {strong_}{closedWith}{_strong}.', args)
+        : L("{strong_}{name}'s{_strong} proposal to add {member} to the group was {strong_}{closedWith}{_strong}.", args),
+      [PROPOSAL_REMOVE_MEMBER]: isCreator
+        ? L('your proposal to remove {member} from the group was {strong_}{closedWith}{_strong}.', args)
```

Comment:
Every sentence that starts with "your" needs to be replaced with "Your" because sentences should start with an uppercase letter.